doc/source/tech_note/Crop_Irrigation/CLM50_Tech_Note_Crop_Irrigation.rst

```diff
@@ -535,24 +543,125 @@ fulfill plant nitrogen demands.
 Harvest
 ''''''''''''''''''''''''''''''
 
-Variables track the flow of grain C and
-N to food and of all other plant pools, including live stem C and N, to litter. Putting live
-stem C and N into the litter pool is in contrast to the approach for unmanaged PFTs which
-puts live stem C and N into dead stem pools first. Leaf and root C and N pools
-are routed to the litter pools in the same manner as natural vegetation. Whereas food C and N
-was formerly transferred to the litter pool, CLM5 routes food C and N
+Variables track the flow of grain C and N to food and of all other plant pools, including live stem C and N, to litter, and to biofuel feedstock.
+A fraction (determined by :math:`biofuel\_harvfrac`) of leaf/livestem C and N from bioenergy crops is removed at harvest for biofuels 
+(Equations :eq:`25.9`, :eq:`25.10`, :eq:`25.12`, and :eq:`25.13`),
+with the remaining portions go to the litter pools (Equations :eq:`20.14)`, :eq:`25.11`, and :eq:`25.14`).
```

Comment:
Please rephrase to "... with the remaining portions going to the litter pools..."